docs/custom_filter.md

```diff
@@ -0,0 +1,103 @@
+Custom Filter
+=============
+
+Sylius Grids come with built-in filters, but there are use-cases where you need something more than basic filter. Grids allow you to define your own filter types!
+
+To add a new filter, we need to create an appropriate class and form type.
+
+```php
+    <?php
```

Comment:
This is the only file, where whole PHP and YAML content has indents.   
```suggestion
<?php
```

can you apply this to the rest of file?